### PR TITLE
fix(components/ag-grid): remove hover states for read-only data grid

### PR DESCRIPTION
**Pull request by Blackbaud-SandhyaRajasabeson**
[#AB2392130](https://blackbaud.visualstudio.com/Products/_boards/board/t/SKY%20UX%20Program/Stories/?workitem=2392130)
[#AB2455803](https://blackbaud.visualstudio.com/Products/_boards/board/t/SKY%20UX%20Program/Stories/?workitem=2455803)

### CODE REVIEW

File: libs/components/ag-grid/src/lib/styles/_modern-dark.scss [hunk 1]
```diff
@@ -3,6 +3,10 @@
 
 .ag-theme-sky-modern-dark {
   ag-grid-angular:not(.sky-ag-grid-editable) {
+    .ag-row-hover::before {
+      background-color: transparent;
+    }
+
```
Comment (Blackbaud-SandhyaRajasabeson) [inline libs/components/ag-grid/src/lib/styles/_modern-dark.scss:9]: angular materials adds a background color when hovering over a row but only in dark mode, is there a better way to remove that?

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1104/storybook/)

Component Storybooks:

- [a11y-storybook](https://blackbaud.github.io/skyux-pr-preview/1104/storybooks/a11y-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1104/storybooks/ag-grid-storybook/)
- [core-storybook](https://blackbaud.github.io/skyux-pr-preview/1104/storybooks/core-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/1104/storybooks/indicators-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/1104/storybooks/layout-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/1104/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/1104/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/1104/storybooks/popovers-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/1104/storybooks/split-view-storybook/)
- [storybook](https://blackbaud.github.io/skyux-pr-preview/1104/storybooks/storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1104/storybooks/tabs-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/1104/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/1104/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1104/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1104/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1104/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1104/playground/)

File: libs/components/ag-grid/src/lib/styles/_modern-light.scss [hunk 2]
```diff
@@ -3,6 +3,10 @@
 
 .ag-theme-sky-modern-light {
   ag-grid-angular:not(.sky-ag-grid-editable) {
+    .ag-row-hover::before {
+      background-color: transparent;
+    }
+
```
Comment (Blackbaud-SandhyaRajasabeson) [inline libs/components/ag-grid/src/lib/styles/_modern-light.scss:9]: future proofing light mode jic angular adds a background color for row hover in light mode in the future

**Review by johnhwhite — APPROVED** [review]

**Comment by blackbaud-sky-build-user** [comment]
Cherry-pick [failed](https://github.com/blackbaud/skyux/actions/runs/4377263704)! :x: Please resolve conflicts and create a new PR.

**Pull request merged**